src/PrestaShopBundle/Resources/config/services/adapter/module.yml

```diff
@@ -64,3 +73,6 @@ services:
 
   prestashop.adapter.module.repository.module_repository:
     class: 'PrestaShop\PrestaShop\Adapter\Module\Repository\ModuleRepository'
+    arguments:
+      - '@=service("prestashop.adapter.legacy.configuration").get("_PS_ROOT_DIR_")'
+      - '@=service("prestashop.adapter.legacy.configuration").get("_PS_MODULE_DIR_")'
```

Comment:
Same here, it's deprecated to call `get()` without a shop constraint

Reply:
Fixed. I remove the call to `prestashop.adapter.legacy.configuration`. `_PS_ROOT_DIR_` &`_PS_MODULE_DIR_` are not constants, not configurations.